### DISCUSSION TITLE
PWAsForFirefox 2.6.2

### DISCUSSION THREAD

**Discussion by filips123** [filips123/PWAsForFirefox#362] [Announcements]
## Added

* Scoop and Gentoo to a list of available repositories.
* Ability to locate 7-Zip in the PATH environment variable (#358).
* Project information in the extension settings (#276).

## Changed

* Bump openssl to fix potential security problem (#360, thanks to @Interpause). 

## Fixed

* The icon bar is now centered again on macOS.
* The icon bar is now vertically centered when there are no icons (#359).

<hr /><em>This discussion was created from the release <a href='https://github.com/filips123/PWAsForFirefox/releases/tag/v2.6.2'>PWAsForFirefox 2.6.2</a>.</em>

**Comment by miroslav-suvada** [comment]
Can't upgrade with winget
```
$ winget install firefoxpwa --version 2.6.2
Found an existing package already installed. Trying to upgrade the installed package...
No version found matching: 2.6.2
```

**Reply by filips123** [reply]
It hasn't been merged yet to the winget repository: microsoft/winget-pkgs#111050

Until it's merged, you can just install the new version from the MSI installer.

**Reply by miroslav-suvada** [reply]
@filips123 ah sure that's what I did. Thanks for the info 👍

**Reply by python-nerd-3** [reply]
you can also just download 2.6.1 if you dont feel like going through another process again
